src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import "./Footer.css"
+const Footer = () => {
+	return (
+		<div>
+			<section class="footer">
+
+				<div class="box-container">
+					<div class="box">
+						<a href="#"> <i class="fab fa-twitter"></i> twitter </a>
```

Comment:
Align the code. You can use prettier for that. Due to improper alignment code is hard to read.